terraform/production/repositories.tfvars

```diff
@@ -14,12 +14,12 @@ repositories = {
   "controls" = {
     description              = "The controls for managing Django Commons projects"
     enable_branch_protection = false
-    allow_merge_commit = true
-    allow_rebase_merge = true
-    allow_squash_merge = true
+    allow_merge_commit       = true
+    allow_rebase_merge       = true
+    allow_squash_merge       = true
     topics = []
     push_allowances = []
-    skip_team_creation = true
+    skip_team_creation       = true
```

Comment:
Is there an auto-formatter we can use for this with something like `pre-commit`? Otherwise I'd be in favor of not doing it.

Reply:
`terraform fmt ...` does it, I'll add to the gha.